jobs/config.json

```diff
@@ -12749,6 +12749,15 @@
       "sig-testing"
     ]
   },
+  "pull-gcp-compute-persistent-disk-csi-driver-test": {
+    "args": [
+      "./test/run-tests.sh"
+    ],
+    "scenario": "execute",
+    "sigOwners": [
+      "sig-cloud-provider-gcp"
```

Comment:
if this is a new sig, add to https://github.com/kubernetes/test-infra/blob/master/jobs/validOwners.json as well to fix the presubmit

Reply:
Done